src/yang/ietf-babel.yang

```diff
@@ -437,50 +453,50 @@ module ietf-babel {
             "RFC ZZZZ: Babel Information Model, Section 3.3.";
         }
 
-	leaf enable {
+        leaf enable {
           type boolean;
           default "true";
           description
-	    "If true, babel sends and receives messages on this
+            "If true, babel sends and receives messages on this
              interface. If false, babel messages received on this
              interface are ignored and none are sent.";
           reference
             "RFC ZZZZ: Babel Information Model, Section 3.3.";
-	}
+        }
 
 	leaf metric-algorithm {
-	  type identityref {
-	    base metric-comp-algorithms;
-	  }
-	  mandatory true;
-	  description
+          type identityref {
+            base metric-comp-algorithms;
+          }
+          mandatory true;
+          description
             "Indicates the metric computation algorithm used on this
-             interface. The value MUST be one of those listed in
-             'metric-comp-algorithms'.";
+             interface. The value MUST be one of those identities
+             based 'metric-comp-algorithms'.";
```

Comment:
s/based/based on/ // I double-checked the email thread to make sure it said "based on"

Reply:
Ok.